interface/main/tabs/main.php

```diff
@@ -107,7 +107,7 @@ function isEncounterLocked( encounterId ) {
 var webroot_url="<?php echo $web_root; ?>";
 </script>
 
-<?php Header::setupHeader(["knockout","tabs-theme"]); ?>
+<?php Header::setupHeader(["knockout","tabs-theme",'jquery-ui','jquery-ui-sunny']); ?>
```

Comment:
Are these even being used anywhere?

Reply:
Used for dialog. The dialog uses top widow for its resting spot for several reason but mainly I kept it there to stay compatible with existing dialog window exits. Jquery-ui has to be in same scope it is being used. ie modal is built in main.

Reply:
Would help to add a comment about that in the code.

Reply:
Do you need the jquery-ui-sunny? Would it make sense to be as minimal as possible and just bring in `base`. If so, I can make this an option for the setupHeader call (I would add support for `jquery-ui-base` call)